### PR TITLE
Enable to configure backup clusters for HR client

### PR DESCRIPTION
**Pull request by mcimbora**

### CODE REVIEW

File: plugins/infinispan52/src/main/java/org/radargun/service/InfinispanServerService.java [hunk 1]
```diff
@@ -57,6 +57,9 @@
    @Property(doc = "Is this executed on Windows (should we use *.bat instead of *.sh)? Default is false.")
    private boolean windows = false;
 
+   @Property(doc = "Name of the script used to start the server. Default is 'clustered'.")
+   protected String startScriptName = "clustered";
```
Comment (rvansa) [inline plugins/infinispan52/src/main/java/org/radargun/service/InfinispanServerService.java]: If this is a property, you'll have to use it for all scripts for Infinispan 8+. I am introducing similar change in the another PR.

**Comment by mcimbora** [comment]
@rvansa Updated, added isnp81 plugin

**Comment by rvansa** [comment]
Besides the need for rebase, I think this can be integrated.

**Comment by mcimbora** [comment]
@rvansa @alanfx Rebased. I also removed jdk18 profile as we'll be using version 8 by default.

**Pull request merged**